assets/js/modules/reader-revenue-manager/components/dashboard/ReaderRevenueManagerSetupCTABanner.test.js

```diff
@@ -81,6 +88,8 @@ describe( 'ReaderRevenueManagerSetupCTABanner', () => {
 	} );
 
 	it( 'should render the Reader Revenue Manager setup CTA banner when not dismissed', async () => {
+		mockSurveyEndpoints();
```

Comment:
We need to retain the `mockSurveyEndpoints` calls to ensure the tests pass, as the survey endpoints are triggered when the banner renders. Removing them would cause the tests to fail.